design-editor/src/system/stage-manager.js

```diff
@@ -259,7 +259,7 @@ class StageManager {
 				}
 			);
 
-			$workSpace.children().first().before(this._previewElementToolbar);
+			$workSpace.children().last().before(this._previewElementToolbar);
 
 			if (isDemoVersion) {
 				const activatePreviewModeButton = document.getElementsByClassName('preview-toggle');
```

Comment:
In Design Editor document there are multiple element with "preview-toggle" class.
Please use more specified selector - since _previewElementToolbar has changed it's position result of this line changed it's order too.
Visibility change is no longer done on the same element.

Reply:
Done!